src/systems.rs

```diff
@@ -163,7 +163,7 @@ pub fn process_input_system(
         win,
     } = *input_resources.modifier_keys_state;
     let mac_cmd = if *context_params.is_macos { win } else { false };
-    let command = if !*context_params.is_macos { win } else { ctrl };
+    let command = if *context_params.is_macos { win } else { ctrl };
```

Comment:
yes, that fix seems correct..

I really wish we could make some tests on that :o ; it's a pain to verify